### DISCUSSION TITLE
BEPINEX does not turn on on Synology.

### DISCUSSION THREAD

**Discussion by Bober-Den** [lloesche/valheim-server-docker#329] [Q&A] [answered]
Provide env variable BEPINEX = true, but  new directory were not created. 
How to fix it?
![image](https://user-images.githubusercontent.com/83207040/116047365-ac314e00-a67c-11eb-9139-172be27e7852.png)

**Comment by lloesche** [comment, marked as answer]
Update your container.
https://github.com/lloesche/valheim-server-docker#updating-the-container-image-to-the-latest-version

**Reply by Bober-Den** [reply]
Thank you! Works!
